repair/repair.cc

```diff
@@ -584,68 +584,76 @@ repair_neighbors shard_repair_task_impl::get_repair_neighbors(const dht::token_r
         neighbors[range];
 }
 
-size_t shard_repair_task_impl::ranges_size() {
+size_t shard_repair_task_impl::ranges_size() const noexcept {
     return ranges.size() * table_ids.size();
 }
 
 // Repair a single local range, multiple column families.
 // Comparable to RepairSession in Origin
-future<> shard_repair_task_impl::repair_range(const dht::token_range& range, ::table_id table_id) {
+future<> shard_repair_task_impl::repair_range(const dht::token_range& range, ::table_id table_id, sstring table_name) {
```

Comment:
Why do we need to pass `table_name`?
`shard_repair_task_impl` keeps reference to `db` and already does:
https://github.com/scylladb/scylladb/blob/941407b905a60923e0b0ac1e6b0399cac2d67b79/repair/repair.cc#L635

Reply:
@Deexie, `table_name` is still passed here, but it is not used.